packages/shared/lib/utils/utils.ts

```diff
@@ -84,11 +84,31 @@ export function parseTokenExpirationDate(expirationDate: any): Date {
         return new Date(expirationDate * 1000);
     }
 
+    // Check for "D+:HH:MM" format (e.g., "177:05:38")(tableau expire in value)
+    if (typeof expirationDate === 'string' && /^\d+:\d{2}:\d{2}$/.test(expirationDate)) {
+        return parseDayHourMinuteDuration(expirationDate);
+    }
```

Comment:
would it be more efficient to try to parse with `new Date()` and fallback to this when it doesn't work?